### PR TITLE
[[ Documentation ]] Updates to LC Script Guide

### PR DESCRIPTION
**Pull request by asayd**
Corrected some typos and completed the Quick Links section, which I hadn't finished in my previous edit.

### CODE REVIEW

File: Documentation/guides/LiveCode Script.md [hunk 1]
```diff
@@ -294,9 +315,13 @@ of links to the most recent posts which have been made.
 
 ##### Blog
 
+This button takes you to LiveCode's online blog, where LiveCode engineers and developers regularly post updates about what's new in LiveCode, how-to tutorials, and announcements of interest to the LiveCode developer community.
```
Comment (montegoulding) [inline Documentation/guides/LiveCode Script.md]: @asayd This and the hunk below should probably be wrapped otherwise this seems good to me

**Comment by peter-b** [comment]
@livecode-vulcan review ok 813ee66

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 813ee66a3e29f1091dcdccdf2c3c9feb822f9b68

**Comment by asayd** [comment]
I want to revise the section on the API dictionary interface, which is badly out of date. Is it best to wait for this PR to be merged before I start another edit? (I.e., edit this patch or start a brand new one?)

**Comment by asayd** [comment]
I went ahead and added the dictionary section revisions to this PR since it hasn't been merged yet.

**Comment by asayd** [comment]
@livecodeali @peter-b Vulcan review has been pending 20+ hours. Hope I didn't mess something up by editing the branch after it was reviewed.

**Comment by peter-b** [comment]
@asayd The review relates to one _specific_ version of the pull request.  If you change the pull request (by adding more commits, for example) it invalidates the review.  This is so you can't do silly things like make a pull request, get it reviewed, and then quickly change out the code so that the wrong changes get merged into the product!

**Comment by peter-b** [comment]
@livecode-vulcan review ok 812b94f

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 812b94f59f6a8718e44e5d6c62bea6ddb13ca665

**Comment by asayd** [comment]
@peter-b Thanks, Peter! I'm keen to get the Script Guide updated, especially screen shots. But due to demands on my time can only do it incrementally. Just wanted to make sure I can't inadvertently mess something up.

**Pull request merged**